kahuna/public/js/components/gr-archiver-status/gr-archiver-status.js

```diff
@@ -29,7 +29,7 @@ archiver.controller('ArchiverCtrl',
     ctrl.undelete = undelete;
 
     mediaApi.getSession().then(session => {
-        if (ctrl.image.data.softDeletedMetadata !== undefined && session.user.permissions.canDelete) { ctrl.canUndelete = true; }
+        if (ctrl.image.data.softDeletedMetadata !== undefined && session.user.permissions.canDelete || session.user.email === ctrl.image.data.uploadedBy) { ctrl.canUndelete = true; }
```

Comment:
I think this (and the other conditions like this in this latest commit) needs to have brackets around the or, like this:

`if (ctrl.image.data.softDeletedMetadata !== undefined && (session.user.permissions.canDelete || session.user.email === ctrl.image.data.uploadedBy)) { ctrl.canUndelete = true; }`

without them, the condition evaluates to true even when the image isn't soft deleted - and I can see an 'Undelete' button when uploading an image for the first time!

![image](https://user-images.githubusercontent.com/10963046/134198718-72fbb941-49ac-4fb9-80ca-8903bbdf5331.png)